docs/config.rst

```diff
@@ -380,8 +380,7 @@ Implementation
 
         @Override
         public Raven createRavenInstance(Dsn dsn) {
-            Raven raven = new Raven();
-            raven.setConnection(createConnection(dsn));
```

Comment:
Yeah, I think this should be fine since I think the parameterless constructor was the only place that it could justifiably be used.